sdk/communication/azure-communication-networktraversal/README.md

```diff
@@ -1,144 +1,10 @@
 # Azure Communication Network Traversal Package client library for Python
 
-Azure Communication Network Traversal is managing TURN credentials for Azure Communication Services.
-
-It will provide TURN credentials to a user.
-
-[Source code](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication) | [API reference documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication)
-
-## _Disclaimer_
-
-_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
-
-# Getting started
-
-### Prerequisites
-
-- Python 3.7 or later is required to use this package.
-- You must have an [Azure subscription](https://azure.microsoft.com/free/)
-- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
-
-### Install the package
-
-Install the Azure Communication Identity client library for Python with [pip](https://pypi.org/project/pip/):
-Install the Azure Communication Relay Client library for Python with [pip](https://pypi.org/project/pip/):
-
-```bash
-pip install azure-communication-identity
-pip install azure-communication-networktraversal
-```
-
-# Key concepts
+Please note, this package has been deprecated and will no longer be maintained after 03/31/2024.
```

Comment:
I don't think we should remove all the content from the readme. Perhaps just add this note to the intro and modify the  "Getting Started" section. I would leave the rest intact.